src/Bootstraps/DiscoveryLocationBootstrap.php

```diff
@@ -6,18 +6,15 @@
 
 use Tempest\AppConfig;
 use Tempest\Application\Kernel;
-use Tempest\Container\Container;
 use Tempest\Discovery\DiscoveryLocation;
 use function Tempest\path;
 
 final readonly class DiscoveryLocationBootstrap implements Bootstrap
 {
-    private string $root;
-
     public function __construct(
-        private Container $container,
+        private AppConfig $appConfig,
```

Comment:
I did some code cleanup in this file, it's not entirely related to this PR, but I stumbled across it when debugging some issues for this PR, so I include it here.